watertap/examples/flowsheets/MD/MD_single_stage_continuous_recirculation.py

```diff
@@ -0,0 +1,662 @@
+    iscale.set_scaling_factor(m.fs.heater.control_volume.heat, 1e-3)
+    iscale.set_scaling_factor(m.fs.chiller.control_volume.heat, 1e-3)
+    iscale.set_scaling_factor(m.fs.MD.area, 1e-1)
+
+    iscale.calculate_scaling_factors(m)
+
+    return m
+
+
+def add_costs(m):
+    m.fs.costing = WaterTAPCosting()
+    m.fs.MD.costing = UnitModelCostingBlock(flowsheet_costing_block=m.fs.costing)
+    m.fs.heater.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=cost_heater_chiller,
+        costing_method_arguments={"HC_type": "electric_heater"},
+    )
+    m.fs.chiller.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=cost_heater_chiller,
+        costing_method_arguments={"HC_type": "chiller"},
+    )
+    m.fs.pump_feed.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=cost_pump,
+        costing_method_arguments={"pump_type": "low_pressure"},
+    )
+    m.fs.pump_permeate.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=cost_pump,
+        costing_method_arguments={"pump_type": "low_pressure"},
+    )
+    m.fs.pump_brine.costing = UnitModelCostingBlock(
+        flowsheet_costing_block=m.fs.costing,
+        costing_method=cost_pump,
+        costing_method_arguments={"pump_type": "low_pressure"},
+    )
+    m.fs.hx.costing = UnitModelCostingBlock(flowsheet_costing_block=m.fs.costing)
+    m.fs.mixer.costing = UnitModelCostingBlock(flowsheet_costing_block=m.fs.costing)
+
+    m.fs.costing.cost_process()
+    m.fs.costing.add_annual_water_production(m.fs.permeate.properties[0].flow_vol)
+    m.fs.costing.add_specific_energy_consumption(m.fs.permeate.properties[0].flow_vol)
+    m.fs.costing.add_LCOW(m.fs.permeate.properties[0].flow_vol)
+
+
+def set_operating_conditions(m):
+    # overall recovery
+    m.fs.overall_recovery.fix(0.5)
+    # feed
+    feed_flow_mass = 1
+    feed_mass_frac_TDS = 0.035
+    feed_pressure = 101325  # atmospheric
+    feed_temperature = 273.15 + 25
+    feed_mass_frac_H2O = 1 - feed_mass_frac_TDS
+    m.fs.feed.properties[0].flow_mass_phase_comp["Liq", "TDS"].fix(
+        feed_flow_mass * feed_mass_frac_TDS
+    )
+    m.fs.feed.properties[0].flow_mass_phase_comp["Liq", "H2O"].fix(
+        feed_flow_mass * feed_mass_frac_H2O
+    )
+
+    m.fs.feed.properties[0].pressure.fix(feed_pressure)
+    m.fs.feed.properties[0].temperature.fix(feed_temperature)
+
+    # concentrate separator
+    # either the overall recovery or the split fraction will be fixed, but not both
+    recycle_fraction = 0.9
+    recycle_flow_mass = 10 * feed_flow_mass
+    # MD
+    membrane_pressure_drop = -5e5
+    membrane_area = 10 * recycle_flow_mass
+    m.fs.MD.area.fix(membrane_area)
+    m.fs.MD.permeability_coef.fix(1e-10)
+    m.fs.MD.membrane_thickness.fix(1e-4)
+    m.fs.MD.membrane_tc.fix(0.2)
```

Comment:
I would rename this something like `membrane_therm_cond` or even the longer `membrane _thermal_conductivity` for better readability. Would need to update in documentation too

Reply:
addressed